src/Compilers/CSharp/Test/Semantic/Semantics/UninitializedNonNullableFieldTests.cs

```diff
@@ -235,6 +254,27 @@ public void FieldInitializers_DefaultConstructor()
             comp.VerifyDiagnostics();
         }
 
+        [Fact]
+        [WorkItem(34668, "https://github.com/dotnet/roslyn/issues/34668")]
+        public void StaticFields_DefaultConstructor_NoInitializers()
+        {
+            var source =
+@"class C
+{
+#pragma warning disable 0169
+    private static object F;
+    private static object P { get; set; }
```

Comment:
We don't warn about uninitialized field-like events, right?

Reply:
We do. See `StaticEvents` test below.

---
In reply to: [307936169](https://github.com/dotnet/roslyn/pull/37507#discussion_r307936169) [](ancestors = 307936169)